tslib/src/functions/school-pin-generator/school-pin-replenishment.data.service.ts

```diff
@@ -16,11 +16,20 @@ export class SchoolPinReplenishmentDataService implements ISchoolPinReplenishmen
 
   async getSchoolById (id: number): Promise<School | undefined> {
     const sql = `
-    SELECT s.id, s.name,  s.pinExpiresAt, s.pin, sce.timezone
-    FROM mtc_admin.school s
-    LEFT OUTER JOIN mtc_admin.sce ON s.id = sce.school_id
-    WHERE s.id = @school_id AND
-    (s.pinExpiresAt <= GETUTCDATE() OR s.pinExpiresAt IS NULL)`
+    SELECT
```

Comment:
my PR touches the same files so will have to take care when merging this into mine

Reply:
This is only a formatting change, but yes, we should be careful. 👀